task/task_smoke_test.go

```diff
@@ -40,9 +89,17 @@ func TestTaskSmoke(t *testing.T) {
 	}
 
 	if testName == "" {
-		testName = "smoke test"
+		testName = fmt.Sprintf("smoke test: %s", targetName)
 	}
-
+	script := `
+          #!/bin/sh -e
+          [ -n "$TEST_GPU" ] && nvidia-smi
+          mkdir --parents cache output
+          touch cache/file
+          echo "$ENVIRONMENT_VARIABLE_DATA" | tee --append output/file
+          sleep 60
+          cat output/file
+        `[1:]
```

Comment:
If you really want to keep this script outside, please consider surrounding it with newlines.

```suggestion

	script := `
          #!/bin/sh -e
          [ -n "$TEST_GPU" ] && nvidia-smi
          mkdir --parents cache output
          touch cache/file
          echo "$ENVIRONMENT_VARIABLE_DATA" | tee --append output/file
          sleep 60
          cat output/file
        `[1:]

```

Reply:
Updated. I think it makes it more readable to define the script outside the multiple levels of indentation in the task definition.